app/styles/layout.scss

```diff
@@ -2,29 +2,15 @@
   display: flex;
   flex-direction: column;
   border-radius: 8px;
-
-  .nav-button {
-    color: black;
-    border-radius: 8px;
-    border: 1px solid #1890ff;  // Border color same as text color
-    padding: 10px;  // Adjust as needed
-    margin-bottom: 8px;  // Adjust as needed
-
-    &:hover {
-      background-color: #d4e4ff;  // Darken the background color on hover
-    }
-  }
-}
-
-.content {
-  display: flex;
-  flex-direction: row;
-  margin-top: 40px;
 }
```

Comment:
Keep using .content margin-top: 40px. Otherwise the content will be too close to header.